packages/v4/patternfly-docs/content/design-guidelines/components/toolbar/toolbar.md

```diff
@@ -37,7 +37,22 @@ A search filter is intended to be used with a filter group as shown below when y
 ### Button group
 <img src="./img/button-group.png" alt="button group" width="205" />
 
- Any global actions that apply to selected data items are included left-aligned following filter and icon buttons. No more than two items should be exposed as buttons. If you have more than two items, use an [overflow menu](#overflow-menu) component to save space.
+Any global actions that apply to selected data items are included left-aligned following filter and icon buttons. Be selective about which actions to surface on the toolbar, exposing no more than two buttons and reserving the rest for an [overflow menu](#overflow-menu) component to save space.
+
+ **One or two clickable actions**
+ <img src= "./img/primary-button.png" alt= "single button" width="196"/>
+* Use a primary button for any call to action you want a user to take
+* In the case that using the action represented should not be encouraged, opt for a secondary button
+
+ **Three clickable actions**
+<img src= "./img/three-as-a-group.png" alt= "three actions" width="196"/>
```

Comment:
add line of space between bolded text and images in all these images, and update width to be correct